<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SciFi</class>
 <widget class="QWidget" name="SciFi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sci-Fi</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../../../Downloads/libro.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#107acc
;</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Sci-Fi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Choose a Theme</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="spaceBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#289ccc;
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Space Exploration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/spacexploration.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/alieninvasion.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="alienBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#289ccc;
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Alien Invasion</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
}
QPushButton:pressed{
background-color:#127bbc;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/arrow.png</normalon>
    </iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
